src/pages/index.js

```diff
@@ -91,8 +91,9 @@ const Home = ({ data }) => {
                   size="lg"
                   variant="link"
                   className="mt-4 py-0 fw-semibold text-decoration-none text-primary"
-                  target="_blank"
-                  href={urls?.specRepo || ""}
+                  // target="_blank"
+                  href={ROUTES.contribute.route || ""}
+                  // href={urls?.specRepo || ""}
```

Comment:
let's just remove them
```suggestion
                  href={ROUTES.contribute.route || ""}
```